config/src/main/java/org/axonframework/config/EventProcessingModule.java

```diff
@@ -252,15 +257,33 @@ private void registerSimpleEventHandlerInvokers(
         assignments.forEach((processingGroup, handlers) -> {
             String processorName = processorNameForProcessingGroup(processingGroup);
             handlerInvokers.computeIfAbsent(processorName, k -> new ArrayList<>()).add(
-                    c -> SimpleEventHandlerInvoker.builder()
-                                                  .eventHandlers(handlers)
-                                                  .handlerDefinition(retrieveHandlerDefinition(handlers))
-                                                  .parameterResolverFactory(configuration.parameterResolverFactory())
-                                                  .listenerInvocationErrorHandler(listenerInvocationErrorHandler(
-                                                          processingGroup
-                                                  ))
-                                                  .sequencingPolicy(sequencingPolicy(processingGroup))
-                                                  .build()
+                    c -> {
+                        if (deadLetterQueues.containsKey(processingGroup)) {
+                            return DeadLetteringEventHandlerInvoker.builder()
+                                                                   .eventHandlers(handlers)
+                                                                   .handlerDefinition(retrieveHandlerDefinition(handlers))
+                                                                   .parameterResolverFactory(configuration.parameterResolverFactory())
+                                                                   .sequencingPolicy(sequencingPolicy(processingGroup))
+                                                                   .queue(deadLetterQueue(processingGroup))
+                                                                   .processingGroup(processingGroup)
+                                                                   .transactionManager(transactionManager(processorName))
+                                                                   .listenerInvocationErrorHandler(
+                                                                           listenerInvocationErrorHandler(
```

Comment:
Retrieving the `ListenerInvocationErrorHandler` for a `processingGroup` will mean we set the default `LoggingErrorHandler`. This will, however, ensure that the `EvalutionTask` automatically acknowledges a failed dead letter. Should we allow a second `ListenerInvocationErrorHandler` configuration? Or disrgard this configuration's field entirely?

Reply:
I think the two should be mutually exclusive. Using a DLQ means you are interested in retrying any error that occurs. Not just once.

We could default this to propagating. However, I feel it should still be configurable since filtering which errors are retried, for example, would be a good case for customization. 

Idea: We could also 'test' the error handler and throw a warning log if the error handler doesn't propagate. We can do this by throwing an (IllegalState|NotImplemented)Exception at it and see whether it rethrows the exception.

Reply:
This idea is as of recent implemented with the so-called `EnqueuePolicy`, which constructs an `EnqueueDecision` based on a `DeadLetter` and the `Throwable` reason for failing event handling.

With that said, I think we can conclude we can remove the `ListenerInvocationErrorHandler` as a configurable component from the `DeadLetteringEventHandlerInvoker` and the `DeadLetterEventProcessingTask`, correct?

Reply:
In absence of response, I assumed removal of the handler to be a fair step to take. As such, it's no longer a configurable part of the `DeadLetteringEventHandlerInvoker`.